app/routes.test.js

```diff
@@ -0,0 +1,402 @@
+import request from 'supertest';
+import { App } from '../app';
+import routes from './routes';
+import * as dashboardControllers from './pages/dashboard/controller';
+import * as sectionControllers from './pages/section/controller';
+import * as subsectionControllers from './pages/dashboard/subDashboards/controller';
+import * as previewControllers from './pages/preview/controller';
+
+jest.mock('./logger');
+
+const mockDashboardContext = {
+  title: 'Dashboard title',
+  mainAdvice: undefined,
+  additionalAdvice: undefined,
+  previewUrl: '/',
+  submitForModerationUrl: '/',
+  returnToDashboardUrl: '/',
+  errors: undefined,
+  sectionGroups: [
+    {
+      id: 'sectiongroup-id',
+      title: 'Sectiongroup Title',
+      sections: [],
+    },
+  ],
```

Comment:
If some of the keys are undefined. Do we need to still explicitly set this? I think we could get away with just removing them.

Reply:
I just copied them from real contexts that came back from requests but I have removed them and checked it still works 👍